### ISSUE TITLE
Limiting CMC When Searching at Random?

### ISSUE THREAD

**Issue by YouSoSully** [MagicTheGathering/mtg-api#47]
Is there a way to set a max CMC or input multiple values for a possible CMC when searching at Random?
It looks like currently colors, supertypes, and subtypes are the only fields that allow multiple values.
For instance say I want to fetch a random equipment (think MoJhoSto) and have it be 3 CMC or less, can I achieve this through cmc=3|2|1|0?

Or is there a way to accomplish that I can't seem to figure out?

**Comment by WPaul15** [comment]
I don't think that has anything to do with including `random=true` in the query params. It looks like `cmc` currently only supports a single value.
